src/api/apps/articles/model/save.coffee

```diff
@@ -12,7 +12,6 @@ request = require 'superagent'
 Article = require './index'
 { distributeArticle, deleteArticleFromSailthru, getArticleUrl, indexForSearch } = require './distribute'
 { ARTSY_URL, GEMINI_CLOUDFRONT_URL } = process.env
-artsyXapp = require('artsy-xapp').token or ''
```

Comment:
I don't think there's ever a need for this default of `''`.

Reply:
Aha! There's a spec failure that errors b/c `undefined` isn't a valid value for a header. So that's why someone added this default 😢

Reply:
@alloy I also think what might fix this, is to leave `artsyXapp = require('artsy-xapp')` at the top here (but remove the `.token` part), and do that in line. So the 'bug' was trying to read the actual token itself here, vs requiring the module.

Reply:
Aye 👍

Reply:
Yup, did indeed fix it. I think that's preferable then.